vault/activity_log.go

```diff
@@ -2166,13 +2166,8 @@ func (a *ActivityLog) precomputedQueryWorker(ctx context.Context) error {
 		for nsID, entry := range byNamespace {
 			mountRecord := make([]*activity.MountRecord, 0, len(entry.Mounts))
 			for mountAccessor, mountData := range entry.Mounts {
-				valResp := a.core.router.ValidateMountByAccessor(mountAccessor)
-				if valResp == nil {
-					// Only persist valid mounts
-					continue
```

Comment:
I want to verify that this is the correct behavior - it seems wrong to me that these currently get thrown out, but I definitely might be missing something

Reply:
and to clarify, deleted mounts were being excluded from the per-mount breakdown of counts. They were still present in the top-level totals.